code/modules/mob/dead/observer/observer_base.dm

```diff
@@ -477,6 +478,16 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 	orbit_menu.ui_interact(src)
 
+/mob/dead/observer/verb/crew_monitor()
+	set category = "Ghost"
+	set name = "Crew Monitor"
+	set desc = "Use a ghastly crew monitor that lets you follow people you select."
+
+	if(!crew_monitor)
+		crew_monitor = new(src)
```

Comment:
It seems this is never deleted when an observer is destroyed?

Reply:
addressed, it's now deleted like the orbit menu